src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php

```diff
@@ -105,7 +104,6 @@ protected function createConfigurationDefinition()
         $annotations = new FixerOptionBuilder('annotations', 'List of annotations to remove, e.g. `["author"]`.');
         $annotations = $annotations
             ->setAllowedTypes(array('array'))
-            ->setDefault(array())
```

Comment:
yes, as any of General fixer behave. we already have few non-general fixers that approves empty configs and skip the fixing.
the only rule that **requires** configuration is header, as we cannot guess what header comment shall be there.